legacy-fabric-resource-loader-v1/build.gradle

```diff
@@ -1 +1,5 @@
 version = getSubprojectVersion(project, "1.0.0")
+
+moduleDependencies(project, [
+		'legacy-fabric-logger-api-v1'
```

Comment:
Rather than doing this for every module, add it somewhere in build.gradle so that every su module gets the logger module. Easier. Upstream does that for game test (and thus resource loader as well)

Reply:
done